stone-co/ex-pix-brcode
# Fix: Include additional validations to compliance

The main goal is include additional validations according BACEN tester software uses cases following the specification [EMV® QR Code Specification for Payment Systems (EMV® QRCPS) Merchant-Presented Mode](https://www.emvco.com/emv-technologies/qrcodes/). 

Validations can be summarize as follows:
* QR Code size respecting 512 bytes maximum
* Alphanumeric Special format for fields that follow it
* Remove zero acceptance for transaction amount
* UUID format for random key
* Ensure no protocol presence on URL
